Test/TestCases/A/Another.cs

```diff
@@ -2,40 +2,42 @@
 
 namespace NodeApi.Examples;
 
+#pragma warning disable CA1822 // Mark members as static
+
```

Comment:
Should we suppress all code analysis for test modules?

Or we can suppress this specific warning during the build, by adding `NoWarn="CA1822"` property in `BuildTestModuleCSharp()`.

Reply:
`dotnet format` was actually changing instance methods in this class to static methods. It was causing the test to break.
Maybe in real code it should not be an issue because instance methods supposed to access the instance variables.
Let's start writing real tests before deciding to ignore this warning for all test modules.